dir_class.py

```diff
@@ -159,3 +160,15 @@ def reading_data_from_text_file(self, file_name):
                 elif action == "u":
                     os.remove(file_path)
 
+    def new_file_name(self, file_name):
+        file_path = self.base_dir + os.sep + file_name
+        audio = EasyID3(file_path)
+        file_title = audio["title"]
+        if audio["title"] == "":
+            file_title = "-"
+        file_tracknumber = audio["tracknumber"]
+        if audio["tracknumber"] == "":
+            file_tracknumber = "-"
```

Comment:
Jeżeli title, tracknumber są puste, to wolę żeby te pliki się nie przenosiły, niż żeby wszystkie zwaliły mi się w jedno miejsce pod nazwą "--".